### PR TITLE
MCR-1725 Improve Datacite registration

### PR DESCRIPTION
**Pull request by yagee-de**

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 122bf592724ed45db3b15b76bfe9b2758b91657e! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/2189-pull_request-401

File: mycore-pi/src/main/resources/components/pi/config/mycore.properties [hunk 1]
```diff
@@ -41,10 +41,8 @@ MCR.PI.Generator.MapObjectIDDOI=org.mycore.pi.doi.MCRMapObjectIDDOIGenerator
 # MCR.PI.Generator.DNBDateURN.Pattern=yyyyMMdd-HHmmss-
 
 # Inscribers
-MCR.PI.Inscriber.MODSDOI=org.mycore.mods.identifier.MCRMODSDOIPersistentIdentifierMetadataManager
```
Comment (yagee-de) [inline mycore-pi/src/main/resources/components/pi/config/mycore.properties:44]: accidently removed?

File: mycore-pi/src/main/resources/components/pi/config/mycore.properties [hunk 2]
```diff
@@ -41,10 +41,8 @@ MCR.PI.Generator.MapObjectIDDOI=org.mycore.pi.doi.MCRMapObjectIDDOIGenerator
...
+# MCR.PI.MetadataManager.DefaultURNXpath=org.mycore.pi.urn.MCRURNObjectXPathMetadataManageranager
+# MCR.PI.MetadataManager.DefaultURNXpath.Xpath=/mycoreobject/metadata/def.identifier[@class='MCRMetaLangText']/identifier[@type='urn']
 
-# MCR.PI.Inscriber.DefaultURNXpath=org.mycore.pi.urn.MCRURNObjectXPathMetadataManageranager
-# MCR.PI.Inscriber.DefaultURNXpath.Xpath=/mycoreobject/metadata/def.identifier[@class='MCRMetaLangText']/identifier[@type='urn']
 
-# MCR.PI.Inscriber.DefaultModsURN=org.mycore.mods.identifier.MCRMODSURNPersistentIdentifierMetadataManager
```
Comment (yagee-de) [inline mycore-pi/src/main/resources/components/pi/config/mycore.properties:49]: accidently removed?

Comment (sebhofmann) [reply mycore-pi/src/main/resources/components/pi/config/mycore.properties:49]: You can find it in mycore-mods.

Comment (sebhofmann) [reply mycore-pi/src/main/resources/components/pi/config/mycore.properties:49]: But there is still a Bug.

**Pull request merged**